### ISSUE TITLE
Exception thrown when running macOS tests

### ISSUE THREAD

**Issue by cyrusingraham** [net-a-porter-mobile/XCTest-Gherkin#159]
This issue seems to effect all new macOS test targets. I have found the issue and will have a pull request open shortly. I am creating the issue to explain why the change is needed.

macOS 10.15.3, Xcode 11.3.1

**Steps to reproduce**
1. Create new macOS application project
<img width="731" alt="create_app-1" src="https://user-images.githubusercontent.com/2578157/77071168-93eefa00-69b9-11ea-8952-f62081b342d5.png">
<img width="727" alt="create_ap-2" src="https://user-images.githubusercontent.com/2578157/77071161-90f40980-69b9-11ea-97b2-9b4fddc799df.png">
2. Create unit test target
<img width="722" alt="create_test_target-1" src="https://user-images.githubusercontent.com/2578157/77071212-a5380680-69b9-11ea-9944-60544d831e22.png">
<img width="722" alt="create_test_target-2" src="https://user-images.githubusercontent.com/2578157/77071221-a6693380-69b9-11ea-9d4a-817369d13a92.png">
3. Add pod file and install

```ruby
platform :osx, '10.15'
use_frameworks!

target 'MyCoolApp' do
  target 'MyCoolAppTests' do
    inherit! :search_paths
    pod 'XCTest-Gherkin'
  end
end
```

4. Create simple example

```swift
import XCTest
import XCTest_Gherkin

class MyCoolAppTests: XCTestCase {
    func testExample() {
        Given("A situation that I want to start at")
        When("I do a thing")
        Then("This value should be 100")
    }
}

class SomeStepDefinitions : StepDefiner {
    override func defineSteps() {
        step("A situation that I want to start at") {
            // Your setup code here
        }
        
        step("I do a thing") {
            // Your setup code here
        }

        step("This value should be ([0-9]*)") { (matches: [String]) in

        }
    }
}
```

5. Run tests

**Expected results**
The test run does not encounter error and the test case passes

**Actual results**
Exception is thrown and test case fails
<img width="1622" alt="exception_thrown" src="https://user-images.githubusercontent.com/2578157/77071576-4aeb7580-69ba-11ea-8c1c-26d5e5f7f116.png">
```
2020-03-19 08:07:08.937969-0500 MyCoolApp[71197:5983892] *** Assertion failure in +[UINSServiceViewController initialize], /BuildRoot/Library/Caches/com.apple.xbs/Sources/ViewBridge/ViewBridge-464.1/UINSServiceViewController.m:203
MyCoolApp/MyCoolAppTests/MyCoolAppTests.swift:6: error: -[MyCoolAppTests.MyCoolAppTests testExample] : failed: caught "NSInternalInconsistencyException", "UINSServiceViewController requires Marzipan"
(
	0   CoreFoundation                      0x00007fff302718ab __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff66392805 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff3029ad10 +[NSException raise:format:arguments:] + 88
	3   Foundation                          0x00007fff32993241 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 191
	4   ViewBridge                          0x00007fff62aa2d25 +[UINSServiceViewController initialize] + 175
	5   libobjc.A.dylib                     0x00007fff66394985 CALLING_SOME_+initialize_METHOD + 17
	6   libobjc.A.dylib                     0x00007fff663952bc initializeNonMetaClass + 638
	7   libobjc.A.dylib                     0x00007fff66395991 _ZL24initializeAndMaybeRelockP10objc_classP11objc_objectR8mutex_ttILb0EEb + 214
	8   libobjc.A.dylib                     0x00007fff663873db lookUpImpOrForward + 969
	9   libobjc.A.dylib                     0x00007fff66386b99 _objc_msgSend_uncached + 73
	10  libswiftCore.dylib                  0x00007fff66e80c03 swift_dynamicCastObjCClassMetatype + 83
	11  libswiftCore.dylib                  0x00007fff66e2993d _ZL29swift_dynamicCastMetatypeImplPKN5swift14TargetMetadataINS_9InProcessEEES4_ + 93
	12  libswiftCore.dylib                  0x00007fff66e2af79 _ZL22_dynamicCastToMetatypePN5swift11OpaqueValueES1_PKNS_14TargetMetadataINS_9InProcessEEEPKNS_22TargetMetatypeMetadataIS3_EENS_16DynamicCastFlagsE + 505
	13  XCTest_Gherkin                      0x0000000100e42168 $s14XCTest_Gherkin15allSubclassesOfySayxGxlF + 472
	14  XCTest_Gherkin                      0x0000000100e5f48b $s14XCTest_Gherkin0B5StateC20loadAllStepsIfNeededyyF + 235
	15  XCTest_Gherkin                      0x0000000100e62c7a $sSo10XCTestCaseC0A8_GherkinE11performStep_7keyword4file4lineySS_S2SSitF + 522
	16  XCTest_Gherkin                      0x0000000100e64fe2 $sSo10XCTestCaseC0A8_GherkinE11performStep_7keyword4file4lineySS_S2SSitFTo + 194
	17  XCTest_Gherkin                      0x0000000100e6037e $sSo10XCTestCaseC0A8_GherkinE5Given_4file4lineySS_SSSitF + 254
	18  XCTest_Gherkin                      0x0000000100e6044a $sSo10XCTestCaseC0A8_GherkinE5Given_4file4lineySS_SSSitFTo + 138
	19  MyCoolAppTests                      0x0000000100af22f0 $s14MyCoolAppTestsAAC11testExampleyyF + 176
	20  MyCoolAppTests                      0x0000000100af246b $s14MyCoolAppTestsAAC11testExampleyyFTo + 43
	21  CoreFoundation                      0x00007fff301d6fec __invoking___ + 140
	22  CoreFoundation                      0x00007fff301d6e83 -[NSInvocation invoke] + 305
	23  XCTest                              0x000000010033bcc6 __24-[XCTestCase invokeTest]_block_invoke_2.214 + 65
	24  XCTest                              0x00000001003c9cb7 -[XCTMemoryChecker _assertInvalidObjectsDeallocatedAfterScope:] + 51
	25  XCTest                              0x0000000100346bae -[XCTestCase assertInvalidObjectsDeallocatedAfterScope:] + 116
	26  XCTest                              0x000000010033bc56 __24-[XCTestCase invokeTest]_block_invoke.208 + 204
	27  XCTest                              0x00000001003afda0 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
	28  XCTest                              0x00000001003afcbd -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
	29  XCTest                              0x000000010033b707 __24-[XCTestCase invokeTest]_block_invoke + 1120
	30  XCTest                              0x000000010033b200 -[XCTestCase testContextPerformInScope:] + 184
	31  XCTest                              0x000000010033b29a -[XCTestCase invokeTest] + 137
	32  XCTest                              0x000000010033d079 __26-[XCTestCase performTest:]_block_invoke_2 + 43
	33  XCTest                              0x00000001003afda0 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
	34  XCTest                              0x00000001003afcbd -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
	35  XCTest                              0x000000010033cf90 __26-[XCTestCase performTest:]_block_invoke.342 + 88
	36  XCTest                              0x00000001003c1380 +[XCTContext runInContextForTestCase:block:] + 219
	37  XCTest                              0x000000010033c703 -[XCTestCase performTest:] + 668
	38  XCTest                              0x000000010038e8e3 -[XCTest runTest] + 57
	39  XCTest                              0x0000000100336da3 __27-[XCTestSuite performTest:]_block_invoke + 365
	40  XCTest                              0x00000001003364cd -[XCTestSuite _performProtectedSectionForTest:testSection:] + 54
	41  XCTest                              0x00000001003367ca -[XCTestSuite performTest:] + 355
	42  XCTest                              0x000000010038e8e3 -[XCTest runTest] + 57
	43  XCTest                              0x0000000100336da3 __27-[XCTestSuite performTest:]_block_invoke + 365
	44  XCTest                              0x00000001003364cd -[XCTestSuite _performProtectedSectionForTest:testSection:] + 54
	45  XCTest                              0x00000001003367ca -[XCTestSuite performTest:] + 355
	46  XCTest                              0x000000010038e8e3 -[XCTest runTest] + 57
	47  XCTest                              0x0000000100336da3 __27-[XCTestSuite performTest:]_block_invoke + 365
	48  XCTest                              0x00000001003364cd -[XCTestSuite _performProtectedSectionForTest:testSection:] + 54
	49  XCTest                              0x00000001003367ca -[XCTestSuite performTest:] + 355
	50  XCTest                              0x000000010038e8e3 -[XCTest runTest] + 57
	51  XCTest                              0x00000001003db886 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke + 171
	52  XCTest                              0x00000001003db982 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke.84 + 118
	53  XCTest                              0x000000010035d6a7 -[XCTestObservationCenter _observeTestExecutionForBlock:] + 594
	54  XCTest                              0x00000001003db644 -[XCTTestRunSession runTestsAndReturnError:] + 623
	55  XCTest                              0x0000000100319a40 -[XCTestDriver runTestsAndReturnError:] + 458
	56  XCTest                              0x00000001003bd6c8 _XCTestMain + 2473
	57  libXCTestBundleInject.dylib         0x0000000100111962 __copy_helper_block_e8_32s + 0
	58  CoreFoundation                      0x00007fff301f57ab __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
	59  CoreFoundation                      0x00007fff301f56ed __CFRunLoopDoBlocks + 379
	60  CoreFoundation                      0x00007fff301f4731 __CFRunLoopRun + 1257
	61  CoreFoundation                      0x00007fff301f3bd3 CFRunLoopRunSpecific + 499
	62  HIToolbox                           0x00007fff2ed4965d RunCurrentEventLoopInMode + 292
	63  HIToolbox                           0x00007fff2ed492a9 ReceiveNextEventCommon + 356
	64  HIToolbox                           0x00007fff2ed49127 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	65  AppKit                              0x00007fff2d3b9ba4 _DPSNextEvent + 990
	66  AppKit                              0x00007fff2d3b8380 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
	67  AppKit                              0x00007fff2d3aa09e -[NSApplication run] + 658
	68  AppKit                              0x00007fff2d37c465 NSApplicationMain + 777
	69  MyCoolApp                           0x000000010000217d main + 13
	70  libdyld.dylib                       0x00007fff677007fd start + 1
)
```

**Comment by cyrusingraham** [comment]
[Pull Request](https://github.com/net-a-porter-mobile/XCTest-Gherkin/pull/160)

**Comment by kerrmarin** [comment]
Looks like this was merged in back in march and released in 0.19.2. Thanks @cyrusingraham
